Source/ViewerController.swift

```diff
@@ -54,58 +54,58 @@ public class ViewerController: UIViewController {
     /**
      Cache for the reused ViewerItemControllers
      */
-    private let viewerItemControllerCache = NSCache()
+    fileprivate let viewerItemControllerCache = NSCache<NSString, ViewerItemController>()
 
     /**
      Temporary variable used to present the initial controller on viewDidAppear
      */
-    private var initialIndexPath: NSIndexPath
+    fileprivate var initialIndexPath: IndexPath
 
     /**
      The UICollectionView to be used when dismissing and presenting elements
      */
-    private unowned var collectionView: UICollectionView
+    fileprivate unowned var collectionView: UICollectionView
 
     /**
      CGPoint used for diffing the panning on an image
      */
-    private var originalDraggedCenter = CGPointZero
+    fileprivate var originalDraggedCenter = CGPoint.zero
```

Comment:
you can only use `.zero` when variable knows the type it's expecting, since here you're declaring the type, it doesn't know it has to be a CGPoint.